connect/cli/plugins/locale/utils.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Dict, List, Union
+
+import click
+from cmr import render
+
+
+def icon_for_autotranslate(resource: Dict[str, Any]) -> str:
+    return '\u2713' if resource['auto_translation'] else '\u2716'
+
+
+def stats_of_translations(resource: Dict[str, Any]) -> Union[str, int]:
+    count = resource["stats"]["translations"]
+    return count if count else '-'
```

Comment:
you can use `return count or '-'`, it is shorter and clearer, I think

Reply:
Yes you are ritght. Is more readable that way